src/app/(tabs)/_layout.tsx

```diff
@@ -0,0 +1,91 @@
+import { HapticTab } from "@/components/HapticTab";
+import { IconSymbol } from "@/components/ui/IconSymbol";
+import TabBarBackground from "@/components/ui/TabBarBackground";
+import { Colors } from "@/constants/Colors";
+import { useColorScheme } from "@/hooks/useColorScheme";
+import { Tabs, useSegments } from "expo-router";
+import Head from "expo-router/head";
+import React, { useEffect, useState } from "react";
+import { Platform } from "react-native";
+
+const TABS = {
+  index: {
+    title: "Home",
+    id: "index",
+  },
+  query: {
+    title: "Query",
+    id: "query",
+  },
+  socket: {
+    title: "Socket",
+    id: "socket",
+  },
+};
+
+export default function TabLayout() {
+  const colorScheme = useColorScheme();
+  const segments = useSegments();
+  const [title, setTitle] = useState(TABS.index.title);
+  // Compute current title based on tab
+  useEffect(() => {
+    const tab = segments[1] as keyof typeof TABS;
+    setTitle(TABS[tab]?.title);
+  }, [segments]);
+
+  return (
+    <>
+      {Platform.OS === "web" && (
+        <Head>
+          <title>
+            Lunch Hub
+            {title ? `| ${title}` : ""}
+          </title>
```

Comment:
Should we add a space here?
```suggestion
          <title>
            Lunch Hub
            {title ? ` | ${title}` : ""}
          </title>
```

Reply:
I updated in my PR